src/main/java/org/jabref/gui/autocompleter/AutoCompletePreferences.java

```diff
@@ -8,26 +8,24 @@
 
 public class AutoCompletePreferences {
 
+    public enum onlyCompleteNameFormat {
```

Comment:
the prefix `only` doesn't really makes sense, right? Moreover, the enum name should be capitalized.